server/offenceCodeDecisions/Commited.ts

```diff
@@ -0,0 +1,153 @@
+import Decision from './Decision'
+import Question from './Question'
+import Code from './Code'
+
+// Codes
+const todo = new Code('TODO')
+
+function decision(question: Question | string) {
+  return new Decision(question)
+}
+
+export default new Decision()
```

Comment:
This is the data structure for the decisions that need to be made when choosing the offender committed an offence. There need to be 3 more for the other selection on the first page. They are very similar to each other and it would be possible to refactor them all into one instance I think. However I've not done this because:
1) While they are very similar at the moment there is no logical reason that they would be in the future and if they all in one structure having differences will be tricky
2) Refactoring to make all of the structures fit into one makes the data outlined below harder to read.
So instead we'll have 4 instances like this one which more verbose.
I'm not wedded to either approach if anyone has strong feeling happy to go with that!